Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.13.0-rc.1-alpine
+FROM 1.13.3-alpine
```

Comment:
Should you add elixir:1.13.3-alpine @JuanDGiraldoM ??

Reply:
Fixed!